src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,124 @@
+package frc.robot.subsystems;
+
+import java.util.logging.Logger;
+
+import com.ctre.phoenix.CANifier;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.LimitSwitchNormal;
+import com.ctre.phoenix.motorcontrol.LimitSwitchSource;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.RemoteLimitSwitchSource;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.CONSTANTS;
+
+public class Shooter extends SubsystemBase {
+
+    
+    private Logger logger = Logger.getLogger(this.getClass().getName());
+    private double motorSetPoint;
+    private int NumberOfBallsFired = 0;
```

Comment:
Members in Java should be lowerCamelCase, and don't have to be explicitly initialised to 0.